src/components/Pokemon.js

```diff
@@ -0,0 +1,48 @@
+import {useEffect, useState} from "react";
+import axios from "axios";
+import './Pokemon.css'
+
+function Pokemon( {pokemonName} ) {
+
+    const [pokemon, setPokemon] = useState({});
```

Comment:
Als je wilt controleren of het al is ingeladen of niet, dan zou ik een default waarde van `undefined` gebruiken
```suggestion
    const [pokemon, setPokemon] = useState();
```